qiita_pet/handlers/analysis_handlers.py

```diff
@@ -111,6 +111,10 @@ def get(self, analysis_id):
     def post(self, analysis_id):
         analysis_id = int(analysis_id)
         rarefaction_depth = self.get_argument('rarefaction-depth')
+        mdsi = self.get_argument('merge-duplicated-sample-ids', default=False)
+        if mdsi == 'on':
```

Comment:
Should this be changed to: `mdsi = mdsi == 'on'`; so in case that is not equal it goes to `False`? Right now it will have whatever value it had before (i.e. a string - rather than a boolean)